dalibo/temboard
# Provide quickstart out of the box

Closes #174 

### Test instructions

1. `pip install -e .`
1. `docker run --rm -p 5432:5432 postgres`
1. `PGUSER=postgres PGPASSWORD=postgres PGHOST=0.0.0.0 share/create_repository.sh`
1. `PGUSER=temboard PGPASSWORD=temboard PGHOST=0.0.0.0 ./temboard -c share/temboard.conf`
1. Check https://0.0.0.0:8888/. Login, add agent, etc.
